### DISCUSSION TITLE
Is it possible to add this Camera-Based Motion Blur to this library?

### DISCUSSION THREAD

**Discussion by kitaedesigns** [pmndrs/postprocessing#651] [Q&A] [answered]
I found this great camera based Motion Blur effect from Drone World, https://github.com/blaze33/droneWorld/blob/master/src/postprocessing/MotionBlur.js, and was wondering how I could add this to pmndrs/postprocessing or if it was even possible. The code is a bit old now, so I fear it's out of date with the latest versions of Three.js but the effect is great and not as intensive as per-object motion blur. 

You can test it here. https://droneworld.openbloc.com/

**Comment by vanruesc** [comment, marked as answer]
It's definitely possible, but I'm not sure if the quality of this motion blur implementation is good enough to spend time on. I'd rather integrate the best one we can find and maintain that instead of many subpar solutions.
